src/components/item-list.tsx

```diff
@@ -1,6 +1,8 @@
 import { quests } from "../data";
 import { ItemName, ItemSource, QuestProgress } from "../types";
 
+const deadDrop = "💀🗑️";
```

Comment:
We should probably replace this with some kind of google material font icon since it won't load correctly on every device as-is. Not a problem you introduced but just thinking out out.

I might suggest we add the money bag icon here as well for consistency. I think eventually we could also include icons for if things are used for crafting etc, may make sense to house them together.